receiver/mongodbatlasreceiver/config.go

```diff
@@ -47,18 +48,55 @@ type AlertConfig struct {
 	TLS      *configtls.TLSServerSetting `mapstructure:"tls"`
 }
 
+type LogConfig struct {
+	Enabled  bool       `mapstructure:"enabled"`
+	Projects []*Project `mapstructure:"projects"`
+}
+
+type Project struct {
```

Comment:
Can we rename this `ProjectConfig` and name variables of this type something like `projectCfg`, in order to differentiate from variables named `project`? There is a lot of mixed naming and it is confusing.